CONTRIBUTING.md

```diff
@@ -25,30 +19,29 @@ You can [become a contributor by signing the ICLA or by contributing on behalf o
 
 For more information about licensing, copyright, and CLAs see Chef's [Community Contributions](https://docs.chef.io/community_contributions.html) page.
 
-Working with the community
---------------------------
+## Working with the community
 These resources will help you learn more about Chef and connect to other members of the Chef community:
+- [chef](https://discourse.chef.io/) mailing list
+- [#chef](https://botbot.me/freenode/chef) and [#chef-hacking](https://botbot.me/freenode/chef-hacking) IRC channels on irc.freenode.net
 
-* [chef](http://lists.chef.io/sympa/info/chef) and [chef-dev](http://lists.chef.io/sympa/info/chef-dev) mailing lists
-* #chef and #chef-hacking IRC channels on irc.freenode.net
-* [Chef docs](https://docs.chef.io)
-* Chef [product page](https://www.chef.io/chef)
+Also here are some additional pointers to some awesome Chef content:
+- [Chef Docs](https://docs.chef.io/)
+- [Learn Chef](https://learn.chef.io/)
+- Chef [product page](https://www.chef.io/chef)
 
-Overview
---------
+## Overview
 If you're experienced with the toolchain, here are the steps for submitting a patch to Supermarket:
-
 1. [Fork the project](https://github.com/chef/supermarket/fork) on GitHub
```

Comment:
The mixed `ol` and `ul` list items here seem to confuse Markdown. I suggest replacing the numbers with dashes and turn the whole thing into `ul`.